src/commands/get_pseudo_random.rs

```diff
@@ -0,0 +1,36 @@
+//! Get Pseudo Random Bytes
+//!
+//! <https://developers.yubico.com/YubiHSM2/Commands/Get_Pseudo_Random.html>
+//!
+use super::{Command, Response};
+use {CommandType, Connector, Session, SessionError};
+
+/// Get some number of bytes of pseudo random data generated on the device
+pub fn get_pseudo_random<C: Connector>(
+    session: &mut Session<C>,
+    bytes: u16,
+) -> Result<GetPseudoRandomResponse, SessionError> {
+    session.send_encrypted_command(GetPseudoRandomCommand { bytes })
```

Comment:
I'd probably suggest having this return `Result<Vec<u8>, SessionError>` and adding a `.map(|response| response.bytes)` on the end

Reply:
:+1: Do you want that as an extra commit, or a rebase/squash?

Reply:
Probably prefer the latter